integration-tests/tests/snap_find_test.go

```diff
@@ -47,3 +63,50 @@ func (s *searchSuite) TestSearchMustPrintMatch(c *check.C) {
 		c.Check(searchOutput, check.Matches, expected)
 	}
 }
+
+// SNAP_FIND_001: list all packages available on the store
```

Comment:
Just curious about the prefix with SNAP_FIND_001 - is that referencing some test case that was written up somewhere else?